website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -111,40 +111,81 @@ models:
-Redshift-specific configuration includes the typical `dist`, `sort_type`, `sort`, and `backup`.
-For materialized views, there is also the `auto_refresh` setting, which allows Redshift to [automatically refresh](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) the materialized view for you.
-The remaining configuration follows the general [materialized view](/docs/build/materializations#Materialized-View) configuration.
-There are also some limitations that we hope to address in the next version.
+The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html)
+with the following configuration parameters:
 
-### Monitored configuration changes
+| Parameter                 | Type         | Required | Default                | Change Monitoring Support | Reference                                       |
+|---------------------------|--------------|----------|------------------------|---------------------------|-------------------------------------------------|
+| `on_configuration_change` | STRING       | NO       | `'apply'`              | N/A                       |                                                 |
+| `dist`                    | STRING       | NO       | `'EVEN'`               | DROP/CREATE               | [Sortkey / Distkey](#using-sortkey-and-distkey) |
+| `sort`                    | LIST[STRING] | NO       |                        | DROP/CREATE               | [Sortkey / Distkey](#using-sortkey-and-distkey) |
+| `sort_type`               | STRING       | NO       | `'AUTO'` if no `sort`  | DROP/CREATE               | [Sortkey / Distkey](#using-sortkey-and-distkey) |
+|                           |              |          | `'COMPOUND'` if `sort` |                           |                                                 |
+| `auto_refresh`            | BOOLEAN      | NO       | `False`                | ALTER                     | [Auto refresh](#auto-refresh)                   |
+| `backup`                  | BOOLEAN      | HO       | `True`                 | N/A                       | [Backup](#backup)                               |
 
-The settings below are monitored for changes applicable to `on_configuration_change`.
+#### Sample model file:
 
-#### Dist
+<File name='redshift_materialized_view.sql'>
 
-Changes to `dist` will result in a full refresh of the existing materialized view (applied at the time of the next `dbt run` of the model). Redshift requires a materialized view to be
-dropped and recreated to apply a change to the `distkey` or `diststyle`.
+```sql
+{{ config(
+    materialized='materialized_view',
+    on_configuration_change='{ apply | continue | fail }'
+    dist='{ ALL | AUTO | EVEN | <field_name> }',
+    sort=['<field_name>', ...],
+    sort_type='{ AUTO | COMPOUND | INTERLEAVED }'
```

Comment:
Lowercase to match the casing in other examples?
```suggestion
    dist='{ all | auto | even | <field_name> }',
    sort=['<field_name>', ...],
    sort_type='{ auto | compound | interleaved }'
```

Reply:
I was matching the casing in the Redshift docs, though not doing so specifically. I think as far as functionality is concerned, both work (we're largely case-insensitive). And we're not necessarily tied to the platform docs as we regularly mess with configuration before submitting it to the database anyway. In other words, I have no strong feeling either way. Do you know off-hand what the style guide would say?